dev_guide/routes.adoc

```diff
@@ -225,14 +225,16 @@ ifdef::openshift-online[]
 [[custom-route-and-hosts-and-certificates-restrictions]]
 == Restrictions
 
-Custom route hosts are not permitted. Instead, the following host template is
-enforced on all user routes:
+Routes are restricted in {product-title} Starter, but are not restricted in
+{product-title} Pro. Custom route hosts are permitted in {product-title} Pro. If
+using {product-title} Starter, the following host template is enforced on all
+user routes:
 
 ----
 <route-name>-<namespace>.1d35.starter-us-east-1.openshiftapps.com
```

Comment:
This isn't exactly accurate. The "us-east-1" value changes depending on the region the user is provisioned in. I can't think if a good way to generalize this. @luciddreamz Any thoughts?

Reply:
Can we provide this as an example, then give the user a method to determine the route themselves?

Users can create a route, and then `oc get route/<route_name>` to see what their external address is.